INN/umbrella-currentorg
# Updated spacing and font sizes on Submit Listing page

## Changes

This pull request makes the following changes:

- Increase the font sizes on the gray listing section tabs.

Before:
![Screen Shot 2019-07-24 at 4 00 58 PM](https://user-images.githubusercontent.com/18353636/61824645-8d10e700-ae2c-11e9-971d-48fd986beeac.png)

After:
![Screen Shot 2019-07-24 at 3 59 42 PM](https://user-images.githubusercontent.com/18353636/61824648-8d10e700-ae2c-11e9-8f68-d5cd9bc793eb.png)

- Increase size of categories in select2 dropdown.

Before:
![Screen Shot 2019-07-24 at 4 01 05 PM](https://user-images.githubusercontent.com/18353636/61824644-8d10e700-ae2c-11e9-8441-cfa1e21c6f17.png)

After:
![Screen Shot 2019-07-24 at 3 59 50 PM](https://user-images.githubusercontent.com/18353636/61824647-8d10e700-ae2c-11e9-80bb-842790793808.png)

- Increase spacing between fee plan images and Submit A Listing title.

Before:
![Screen Shot 2019-07-24 at 4 00 54 PM](https://user-images.githubusercontent.com/18353636/61824646-8d10e700-ae2c-11e9-96ac-32d9f469b71b.png)

After:
![Screen Shot 2019-07-24 at 3 59 34 PM](https://user-images.githubusercontent.com/18353636/61824649-8d10e700-ae2c-11e9-9767-33bde1c44227.png)

## Why



For #99 

## Testing/Questions

Features that this PR affects:

- Submit a Listing page


Questions that need to be answered before merging:

- [ ] Does it all look ok?

Steps to test this PR:

1. View the Submit a Listing page and check out the 3 sections listed above.
